lib/domUpdates.js

```diff
@@ -4,9 +4,21 @@ const domUpdates = {
    $('.catergory-text').text(`Category: ${game.getCategory()}`);
    $('.round-text').text(`Round: ${game.currentRound}`)
    $('.pop-up-screen').toggle();
-   $('.player1-text').text(game.players[0].name);
-   $('.player2-text').text(game.players[1].name);
-   $('.player3-text').text(game.players[2].name);
+   if ($('.player1-input').val() === '') {
+     $('.player1-text').text('Player1');
+   } else {
+     $('.player1-text').text(game.players[0].name);
+   }
+   if ($('.player2-input').val() === '') {
+     $('.player2-text').text('Player2');
+   } else {
+     $('.player2-text').text(game.players[1].name);
+   }
+   if ($('.player3-input').val() === '') {
+     $('.player3-text').text('Player3');
+   } else {
+     $('.player3-text').text(game.players[2].name);
+   }
   },
```

Comment:
i think this can be refactored with having a more dynamic Player class. If we set the Player class to take in 3 names instead of one, then we can have defaults for each player.  When we instantiate a new player then we can pass argument values by putting them in same order as parameters.